src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -407,6 +413,105 @@ public Response updateDraftVersion( String jsonBody, @PathParam("id") String id,
         }
     }
     
+    @PUT
+    @Path("{id}/addData")
+    public Response addDataToVersion(String jsonBody, @PathParam("id") String id) throws WrappedResponse{
+
+        DataverseRequest req = createDataverseRequest(findUserOrDie());
```

Comment:
From discussion with @sekmiller this boolean should be false

Reply:
fixed